GrocifyApp.UI/src/index.css

```diff
@@ -49,10 +41,6 @@ button {
 button:hover {
   border-color: #646cff;
 }
-button:focus,
-button:focus-visible {
-  outline: 4px auto -webkit-focus-ring-color;
-}
 
 @media (prefers-color-scheme: light) {
```

Comment:
this is the light mode? I mean we have to do something like that for dark mode?

Reply:
I don't know, maybe

Reply:
![image](https://github.com/Catarina98/Grocify/assets/36730010/69f1636e-15b1-4af3-98fb-6743ed485171)


I think we cannot use color--n... in texts, because in dark mode it will give us a lot of trouble, we need to decide the colors for titles and texts, and use the same all over the app, what do you think?